source/releasenotes/2024-08-29-domain-verification.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Diamond and Platinum customers now exempt from domain verification enforcement"
+published_date: "2024-08-29"
+categories: [security, infrastructure]
```

Comment:
Not sure about these categories, but it's what [the last related release note used ](https://docs.pantheon.io/release-notes/2024/07/domain-validation)